dapps/tests/teller-contracts/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "embark-dapp-teller-test-contracts",
```

Not critical, but to be more like the names of the other test dapps:

```suggestion
  "name": "embark-dapp-test-teller-contracts",
```